internal/lookout/ui/src/components/job-sets/ReprioritizeJobSetsOutcome.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { Fragment } from "react"
+
+import {
+  List,
+  ListItem,
+  ListItemText,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@material-ui/core"
+
+import { ReprioritizeJobSetResult } from "../../services/JobService"
+import LoadingButton from "../jobs/LoadingButton"
+
+import "./JobSetActions.css"
+
+type ReprioritizeJobSetsOutcomeProps = {
+  reprioritizeJobSetResult: ReprioritizeJobSetResult
+  isLoading: boolean
+  newPriority: number
+  onReprioritizeJobSets: () => void
+}
+
+export default function ReprioritizeJobSetsOutcome(props: ReprioritizeJobSetsOutcomeProps) {
+  return (
+    <div className="job-sets-action-container">
+      {props.reprioritizeJobSetResult.reprioritizedJobSets.length > 0 && (
+        <Fragment>
+          <p className="cancel-job-sets-text">The following Job Sets were reprioritized successfully:</p>
+          <List component={Paper} className="cancel-job-sets-table-container cancel-job-sets-success">
```

Comment:
These CSS classes have changed name (now in JobSetActions) so should be changed accordingly. This goes for other components which still use `cancel-job-...` classes, including those pertaining to Job Set cancellation